src/lib/chips/_chips-theme.scss

```diff
@@ -13,6 +13,11 @@
   }
 
   .md-chip.selected {
+    background-color: #808080;
```

Comment:
Add comment explaining that this is the same color in both light and dark themes.